v5/releases/cli3.1.4.rst

```diff
@@ -0,0 +1,46 @@
+======================
+PROS CLI 3.1.4 Release
+======================
+
+.. post:: 12 February, 2019
+   :tags: blog, cli-release
+
+- New UI experience for creating, upgrading, and uploading projects
+- Add binary compression to save space on the V5
+- Add support for hot/cold linking
+- Add V5 screen capture with ``prosv5 v5 capture``
+- Add wireless downloading support
+- Added support for supplying upload arguments to ``mu``/``mut``/``ut``
+- Bug fixes to V5 upload protocol
+
+Updating to the new version
+===========================
+
+Python PIP
+----------
+
+Run pip install https://github.com/purduesigbots/pros-cli/releases/download/3.1.4/pros_cli_v5-3.1.4-py3-none-any.whl (or pip3, depending on your system)
+
+Windows
+-------
+
+Download and run the appropriate installer
+
+macOS
+-----
+
+See below for the new recommended method of installing the CLI
```

Comment:
No real need to split this into two sections
```suggestion
To upgrade a previous installation, run ``brew upgrade pros-cli``

To install from scratch:
```
(would need to also delete line 34/35)